.github/workflows/actions.yml

```
@@ -0,0 +1,44 @@
+name: Continuous Integration
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+jobs:
+  specs:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - 2.6.6
+          - 2.7.1
+        gemfile:
+          - gemfiles/rails4.2.gemfile
+          - gemfiles/rails5.0.gemfile
+          - gemfiles/rails5.1.gemfile
+          - gemfiles/rails5.2.gemfile
+        exclude:
+          - {ruby: 2.7.1, gemfile: 'gemfiles/rails4.2.gemfile'} # NoMethodError: undefined method `new' for BigDecimal:Class
+    steps:
+      - uses: zendesk/checkout@v2
+      - uses: zendesk/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - name: test
+        env:
+          BUNDLE_GEMFILE: ${{ matrix.gemfile }}
```

```suggestion
          BUNDLE_GEMFILE: ${{ matrix.gemfile }}
```

I think this line needs to move up one level, e.g. under specs:, not under steps: